src/documentation/index.md

```diff
@@ -40,9 +40,9 @@ A Beam Runner runs a Beam pipeline on a specific (often distributed) data proces
 * [DirectRunner]({{ site.baseurl }}/documentation/runners/direct/): Runs locally on your machine -- great for developing, testing, and debugging.
 * [ApexRunner]({{ site.baseurl }}/documentation/runners/apex/): Runs on [Apache Apex](http://apex.apache.org).
 * [FlinkRunner]({{ site.baseurl }}/documentation/runners/flink/): Runs on [Apache Flink](http://flink.apache.org).
+* [GearpumpRunner]({{ site.baseurl }}/documentation/runners/gearpump/): Runs on [Apache Gearpump (incubating)](http://gearpump.apache.org).
```

Comment:
It seems like we might have to talk about moving to `master` for this :-)

Because of crunch time for the stable release, I have not had a minute to go back and run a few pipelines on real Gearpump. Probably the same is true for other people. We definitely will want to make sure we have the same testing for this runner that we have for others.

It might be fun to bring this up for the community. Maybe it can be part of the discussion of promoting the runner.